pkgs/development/python-modules/omegaconf/default.nix

```diff
@@ -31,6 +31,9 @@ buildPythonPackage rec {
       src = ./antlr4.patch;
       antlr_jar = "${antlr4.out}/share/java/antlr-${antlr4.version}-complete.jar";
     })
+
+    # https://github.com/omry/omegaconf/pull/1137
+    ./0000-add-support-for-dataclasses_missing_type.patch
```

Comment:
Use fetchpatch with includes or excludes instead of vendoring that patch.

Reply:
This is a custom patch on a different base.
The link is just referencing why this patch is necessary.